src/components/header.js

```diff
@@ -51,6 +53,12 @@ export function Header() {
       <Button as={Link} to="/submit-a-talk" sx={{ ml: 2 }}>
         Submit a Talk
       </Button>
+      <NavLink
```

Comment:
`<NavLink>` will renders `a` tag but this is actually a button. Let's use `button` tag because abusing `a` tag as button is not screen-reader friendly. You may need to tweak the styles though.